code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -0,0 +1,212 @@
+/obj/item/weapon/storage/pouch
+	name = "pouch"
+	desc = "Can hold various things."
+	icon = 'icons/inventory/pockets/icon.dmi'
+	icon_state = "pouch"
+	item_state = "pouch"
+
+	w_class = ITEM_SIZE_SMALL
+	slot_flags = SLOT_POCKET
+
+	storage_slots = 1
+	max_w_class = ITEM_SIZE_SMALL
+	max_storage_space = base_storage_capacity(ITEM_SIZE_NORMAL)
+	attack_verb = list("pouched")
+
+	var/sliding_behavior = FALSE
+
+/obj/item/weapon/storage/pouch/verb/toggle_slide()
+	set name = "Toggle Slide"
+	set desc = "Toggle the behavior of last item in [src] \"sliding\" into your hand."
+	set category = "Object"
+
+	sliding_behavior = !sliding_behavior
+	usr << SPAN_NOTICE("Items will now [sliding_behavior ? "" : "not"] slide out of [src]")
+
+/obj/item/weapon/storage/pouch/attack_hand(mob/living/carbon/human/user)
+	if(sliding_behavior && contents.len && (src in user))
+		var/obj/item/I = contents[contents.len]
+		if(istype(I))
+			hide_from(usr)
+			var/turf/T = get_turf(user)
+			remove_from_storage(I, T)
+			usr.put_in_hands(I)
+			add_fingerprint(user)
+	else
+		..()
+
+/obj/item/weapon/storage/pouch/small_generic
+	name = "small generic pouch"
+	desc = "Can hold anything in it, but only about once."
+	icon_state = "small_generic"
+	item_state = "small_generic"
+
+	max_storage_space = ITEM_SIZE_TINY * 2
```

Comment:
Set the storage space of small pouch to ITEM_SIZE_SMALL * 2
and set the storage space of medium pouch to ITEM_SIZE_SMALL * 3

Reply:
That would be slightly too big. And having medium pouch contain 3 small items basically defeats the entire purpose of specialized pouches. I think just how it is is already good.

Reply:
The medium pouch is okay, but the small one is useless, it has almost no benefit. 
Making them double and triple the number of things a pocket can hold, respectively, would be pretty good

I can see that it would cause some issues with the specialised pouches, there's a problem there
Any alternate ideas to make the small pouch more useful?

I'm thinking making crowbars bigger, so that they no longer fit in pockets, and require a pouch or toolbelt to store

Reply:
It is more useful than just a pocket already. It fits two pens instead of one!

Reply:
But nobody uses pens.
Pockets are variably used for...
Flashlight, crowbar, health analyzer, welding tool, syringe, ammo clip

Basically things that you urgently need onhand right now.
Even a small pouch should offer some meaningful advantage by allowing you to store two of these things

Reply:
Also give the janitor a small pouch instead of a medium one, and make the medium pouches generally rarer